MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -226,10 +226,10 @@
 public class Campaign implements Serializable, ITechManager {
     private static final String REPORT_LINEBREAK = "<br/><br/>"; //$NON-NLS-1$
 
-	private static final long serialVersionUID = -6312434701389973056L;
```

Comment:
These Game.java changes are just whitespace formatting, right?

Reply:
Well... that isn't Game.java, but no. Review at the commit level rather than the full change level. I did the whitespace as a separate commit so that the real changes could be seen by looking at commits.